sn_client/src/audit/dag_crawling.rs

```diff
@@ -38,7 +59,7 @@ impl Client {
     pub async fn spend_dag_build_from(
         &self,
         spend_addr: SpendAddress,
-        max_depth: Option<u32>,
+        spend_processing: Option<Sender<SignedSpend>>,
```

Comment:
Optional channel Sender through which we send the Spends collected. 
This is useful for real time external processing of those spends for:
- beta rewards
- following progress
- any other applications (could be used for royalties as well)